README.md

```diff
@@ -27,6 +27,8 @@ Homebrew is a non-profit project run entirely by unpaid volunteers. We need your
 
 Please consider a regular donation through [GitHub Sponsors](https://github.com/sponsors/Homebrew), [Open Collective](https://opencollective.com/homebrew) or [Patreon](https://www.patreon.com/homebrew). Homebrew is fiscally hosted by the [Open Source Collective](https://opencollective.com/opensource).
 
+For questions about donations, including corporate giving, please contact us through our Open Collective [contact form](https://opencollective.com/homebrew/contact).
```

Comment:
Can we use plc@brew.sh, billing@brew.sh, a new address or similar instead here? The OpenCollective form is not great (as demonstrated by the fact we replied to this multiple times without knowing this time)?

Reply:
Certainly.

```suggestion
For questions about donations, including corporate giving, please email the Homebrew PLC at [plc@brew.sh](mailto:plc@brew.sh) or contact us through our Open Collective [contact form](https://opencollective.com/homebrew/contact).
```

Reply:
```suggestion
For questions about donations, including corporate giving, email the Homebrew PLC at [plc@brew.sh](mailto:plc@brew.sh).
```